### PR TITLE
Use dark mode

### LINKED ISSUE #143: Add dark mode
The hugo website had a dark mode which I personally prefer over light mode. Would be good to get that back.

### PR DESCRIPTION
**Pull request by rcannood**
*Transferred from openproblems-bio/website-experimental#107.*

Closes #143 and #138.

### CODE REVIEW

**Review by scottgigante-immunai — COMMENTED** [review]
@KaiWaldrant 's review ported from https://github.com/openproblems-bio/website-experimental/pull/107

File: competitions/neurips_2021_docs/docs.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+img.img-fluid, img.thumnail-image {
```
[the remark this answers is not in the document]
Comment (scottgigante-immunai) [reply competitions/neurips_2021_docs/docs.css:1]: Used at https://theme-dark-mode--openproblems.netlify.app/competitions/neurips_2021_docs/submission/ . Not sure why it's thumnail and not thumbnail, but that's the class name 🤷

[the remark this answers is not in the document]
Comment (KaiWaldrant) [reply competitions/neurips_2021_docs/docs.css:1]: This is fixed in quarto 1.3 quarto-dev/quarto-cli#4602 , so we will need to update it when we switch to  quarto 1.3

**Review by KaiWaldrant — APPROVED** [review]

File: competitions/neurips_2021_docs/docs.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (scottgigante-immunai) [reply competitions/neurips_2021_docs/docs.css:1]: Opened a PR for this. https://github.com/openproblems-bio/website/pull/161

**Review by KaiWaldrant — APPROVED** [review]

**Review by scottgigante-immunai — APPROVED** [review]

**Comment by rcannood** [comment]
@scottgigante-immunai Can you add something to the CHANGELOG.md? :bow:

**Pull request merged**